lms/djangoapps/courseware/field_overrides.py

```diff
@@ -14,6 +14,7 @@
 `LmsFieldData`.  This means overrides will be in effect for all scopes covered
 by `authored_data`, e.g. course content and settings stored in Mongo.
 """
+# pylint: disable=missing-docstring
```

Comment:
@cpennington the book club recently read [Clean Code](http://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882). One of the points Martin (the author) makes is that you shouldn't depend on docstrings and comments to make your code readable. They go stale quickly and turn into clutter.

I'm disabling this because I disagree with Pylint that everything in this module must have a docstring. That said, I intend to respond to @schenedx's comment below by adding a couple.